samples/matter/light_switch/README.rst

```diff
@@ -140,34 +150,44 @@ User interface
     :end-before: matter_door_lock_sample_led1_end
 
 LED 2:
-   Shows the state of the Bluetooth LE Direct Firmware Update (DFU) process.
-
-   * Off - Bluetooth LE is not advertising and DFU is not ready.
-   * Rapid Even Flashing (30 ms off / 170 ms on) - Bluetooth LE is advertising and the DFU process can be started.
-
-LED 3:
    Identifies the device after sending the ``identify`` command to the endpoint ``0`` from the Matter controller device.
    The LED starts blinking evenly (500 ms on/500 ms off) when the Identify command of the Identify cluster is received.
    The command's argument can be used to specify the duration of the effect.
 
-LED 1-4:
+LED 1-4 on nRF52840 DK, nRF5340 DK and nRF21540 DK, or LED 1-2 on nRF7002 DK:
    Blink in unison when the factory reset procedure is initiated.
```

Comment:
```suggestion
LED 1-4 on nRF52840 DK, nRF5340 DK and nRF21540 DK:
   Blink in unison when the factory reset procedure is initiated.
   
LED 1-2 on nRF7002 DK:
   Blink in unison when the factory reset procedure is initiated.
```
I'd say these are two different scenarios, no?
Alternatively, if all LEDs blink:
```suggestion
All LEDs:
   Blink in unison when the factory reset procedure is initiated.
```

Reply:
That's the second case, thanks!.